src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -629,9 +629,9 @@ public static float CalculateDigestionEfficiency(int enzymeCount)
         {
             var organelle = organelles[i];
 
-            // Skip things that don't give anything from endosymbiosis
+            // Skip things that don't give anything from endosymbiosis or don't suit LAWK
             var resultType = organelle.Definition.EndosymbiosisUnlocks;
-            if (resultType == null)
+            if (resultType == null || resultType.LAWK != lawk)
```

Comment:
How does this LAWK check work? Because if I turn off lawk (so lawk = false) and then doesn't this *only* show non-lawk?

I think the right code would be:

```c#
if (!resultType.LAWK && lawk)
   continue;
```

So if organelle is non-LAWK but the game is in LAWK mode, then it should be hidden. But if the organelle is LAWK it doesn't matter what the game's LAWK setting is.